agoric/contract/test/swingsetTests/bootstrap/bootstrap-governance.js

```diff
@@ -0,0 +1,38 @@
+import { AmountMath } from '@agoric/ertp';
+import { makeCopyBag } from '@agoric/store';
+import { E } from '@endo/far';
+import { flow } from '../flow.js';
+
+export async function blockMethods(context) {
+  /** @type {Context} */
+  const { publicFacet, governorFacets, contractAssets, purses, zoe } = context;
+
+  await E(governorFacets.creatorFacet).setFilters(['mintCharacterNfts']);
```

Comment:
I'm wondering how to revert this, maybe `await E(governorFacets.creatorFacet).setFilters([]);`?

Reply:
I havent really looked into this but what you wrote makes sense. It should really affect the tests though cause each test file runs independent of the others.

Reply:
yeah that makes sense, I'll check with Agoric about this